src/cloud-api-adaptor/install/README.md

```diff
@@ -96,21 +112,31 @@ Alternatively the manual approach, if you want to pick a specific CoCo release/r
     ```
   A successful install should show all the PODs with "Running" status under the `confidential-containers-system`
   namespace.
-  
+
     ```
     NAME                                              READY   STATUS    RESTARTS   AGE
     cc-operator-controller-manager-546574cf87-phbdv   2/2     Running   0          43m
     cc-operator-daemon-install-pzc4b                  1/1     Running   0          42m
     cc-operator-pre-install-daemon-sgld6              1/1     Running   0          42m
-    cloud-api-adaptor-daemonset-mk8ln                 1/1     Running   0          37s
+    peerpodconfig-ctrl-caa-daemon-mk8ln                 1/1     Running   0          37s
     ```
 
 * View cloud-api-adaptor logs
 
     ```
-    kubectl logs pod/cloud-api-adaptor-daemonset-mk8ln -n confidential-containers-system
+    kubectl logs ds/peerpodconfig-ctrl-caa-daemon -n confidential-containers-system
     ```
 
+## Workaround: Clean up the images on the worker
```

Comment:
we might not need this anymore due to https://github.com/containerd/nydus-snapshotter/pull/593

Reply:
Yep - once the nydus-snapshotter changes are bumped and included in the operator we'll remove this. Whether we want to wait to merge this until that happens I'm not sure yet.